test/e2e/e2e.go

```diff
@@ -76,6 +78,20 @@ func setupProviderConfig() error {
 			return fmt.Errorf("Error building GCE/GKE provider: %v", err)
 		}
 
+		// TODO(madhusudancs): Remove this once
```

Comment:
That comment is in https://github.com/kubernetes/kubernetes/pull/37571/commits/3806de3752ddd3a86d6ed6740b71613bce4208e7#diff-e00287b5331f6d39959423afadb6b368R134. But also added a comment here.